src/util.rs

```diff
@@ -83,3 +87,38 @@ pub(crate) fn ensure_writable_mount<P: AsRef<Path>>(p: P) -> Result<()> {
     }
     Ok(())
 }
+
+/// Parse the output of `rpm -q`
+pub(crate) fn parse_metadata(stdout: Vec<u8>) -> Result<ContentMetadata> {
```

Comment:
Let's call this `parse_rpm_metadata()` perhaps?  We could also land this as a preparatory PR now!